test/jdk/java/foreign/TestValueLayouts.java

```diff
@@ -145,7 +153,7 @@ void test(ValueLayout layout,
         assertEquals(layout.carrier(), carrier);
         assertEquals(layout.byteSize(), byteSize);
         assertEquals(layout.order(), ByteOrder.nativeOrder());
-        assertEquals(layout.byteAlignment(), byteAlignment);
+        assertEquals(layout.byteAlignment(), (isAixOS && (layout == JAVA_DOUBLE)) ? 32 : byteAlignment);
```

Comment:
Since this deals with _byte_ alignment, `32` should be `4`.

Reply:
Fixed